Sources/Controllers/DashboardOnMap/MapSettings/MapSettingsGpxViewController.swift

```diff
@@ -9,12 +9,81 @@
 import UIKit
 import UniformTypeIdentifiers
 
+private enum TrackSortType {
+    case nearest
+    case lastModified
+    case nameAZ
+    case nameZA
+    case newestDateFirst
+    case oldestDateFirst
+    case longestDistanceFirst
+    case shortestDistanceFirst
+    case longestDurationFirst
+    case shorterDurationFirst
+    
+    var title: String {
+        switch self {
+        case .nearest:
+            return localizedString("shared_string_nearest")
+        case .lastModified:
+            return localizedString("sort_last_modified")
+        case .nameAZ:
+            return localizedString("track_sort_az")
+        case .nameZA:
+            return localizedString("track_sort_za")
+        case .newestDateFirst:
+            return localizedString("newest_date_first")
+        case .oldestDateFirst:
+            return localizedString("oldest_date_first")
+        case .longestDistanceFirst:
+            return localizedString("longest_distance_first")
+        case .shortestDistanceFirst:
+            return localizedString("shortest_distance_first")
+        case .longestDurationFirst:
+            return localizedString("longest_duration_first")
+        case .shorterDurationFirst:
+            return localizedString("shorter_duration_first")
+        }
+    }
+    
+    var iconImage: ImageResource {
```

Comment:
why not ImageResource -> Image?

Reply:
Using ImageResource simplifies accessing image assets within the app. It serves as an abstraction layer that maps each image to a specific case in the enumeration, making it easier to manage and reference images. By encapsulating the image names within ImageResource, we avoid hardcoding string literals throughout the app, which reduces the risk of typos and makes the codebase more maintainable and type-safe.

Reply:
For every reference to you, you need to use the construction = myImageView.image = UIImage(resource: sortType.iconImage), instead of myImageView.image = sortType.image. I don't see much point in storing ImageResource.